### PR TITLE
Update Specviz docs with box zoom explanation

### PR DESCRIPTION
**Pull request by rosteen**
### Description

This pull request is to address ...

Fixes #<Issue Number>

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [ ] Is a milestone set? Milestone is only currently required for PRs related to Imviz MVP.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -117,6 +117,15 @@
 .. |icon-color-square| image:: /img/icons/color_square.png
   :scale: 40
   :alt: color square icon
+
+.. |icon-box-zoom| image:: /img/icons/box_zoom.png
```
Comment (kecnry) [inline docs/conf.py:121]: we really should link directly to the icon images at some point now that almost all are custom... but the rest are screenshots, so let's consider that out-of-scope for now and try to update them all at once at some point in the future?

**Review by kecnry — APPROVED** [review]

File: docs/conf.py (hunk 1 above)
Comment (rosteen) [reply docs/conf.py:121]: Sounds good. I wasn't sure if SVG files would work here so I defaulted to screenshotting for now. Definitely something to explore further to reduce duplicate files.

**Comment by rosteen** [comment]
@kecnry Mind giving the sentences I just added a quick look for spelling, etc? Ready to merge after that I think.

File: docs/cubeviz/displaycubes.rst [hunk 2]
```diff
@@ -104,9 +104,11 @@ Pan/Zoom
 
 Panning and zooming in the Cubeviz viewers works the same as described in the
 section on :ref:`Panning and Zooming in Specviz <pan-zoom>`, with the exception
-that image viewers have only the bidirectional Pan/Zoom tool enabled. The 
-1D spectrum viewer allows panning and zooming along a single axis as in 
-Specviz.
+that image viewers have only the bidirectional Pan/Zoom tool and the two-dimenstional
```
Comment (kecnry) [inline docs/cubeviz/displaycubes.rst:107]:
```suggestion
that image viewers have only the bidirectional Pan/Zoom tool and the two-dimensional
```

**Pull request merged**